woodwork/column_accessor.py

```diff
@@ -357,6 +358,41 @@ def set_semantic_tags(self, semantic_tags):
             _raise_init_error()
         self._schema._set_semantic_tags(semantic_tags)
 
+    def box_plot_dict(self, quantiles=None):
+        """Gets the information necessary to create a box and whisker plot with outliers for a numeric column
+        using the IQR method.
+
+    Args:
+        series (Series): Data for which the box plot and outlier information will be gathered.
+            Will be used to calculate quantiles if none are provided.
```

Comment:
Should be removed since this isn't an argument for this method. Maybe update the quantiles docstring info to indicate they will be computed if not provided.

Reply:
removed the series arg and updated quantiles